docs/docker_run.md

```diff
@@ -0,0 +1,115 @@
+## Setting up a server from a Docker image
```

Comment:
Very nice! Again the test data would be nice here as well

Reply:
👍